### DISCUSSION TITLE
Help Please

### DISCUSSION THREAD

**Discussion by mistanoize** [AnySoftKeyboard/AnySoftKeyboard#2974] [General]
Hi  and thank you for this promising keyboard.

Can you please help me with a few things...

1)  Is there a way to remove/swap the voice input key? I don't use it and have always found it surplus. Currently if I press this key it takes me to settings and offers to download VI from the Google Play Store, so it knows I don't have it. All of which is annoying if you accidentally press it often. 

2)  On the ABC key (top-right key) it shows the name of the keyboard which is really annoying. For example "Dvorak", I would just like to see the traditional "ABC" there. I found the setting to remove the language name from the space bar. Is there a similar setting to remove the keyboard theme name, or can we have this option please?

3)  Is there an option to have 1-0 as standard on the keyboard above the letter keys? If not I would really like to see this in the future please?

I would just like to add I also found the settings non-intuitive, as many others have commented.

Atb.

**Comment by mistanoize** [comment]
TTT :-(

**Comment by mistanoize** [comment]
Is it me or is there no £ symbol? If there is where can I find it please?

**Comment by Jonta** [comment]
@mistanoize I suggest splitting this into different threads, after searching through the issues for duplicates of yours. =)

**Comment by mistanoize** [comment]
@Jonta Thank you for the suggestion. I both searched and scanned the discussion threads before I posted, and tried using the 'top' search box and found nothing. This morning I've been searching Issues and found some useful information, not sure why these results were not returned when using top/main search "https://github.com/search?q=org%3AAnySoftKeyboard+voice+input+key"???

I'm not a developer/coder though I have moderated several forums in the past, and always try to search for what I need before anything else.

**Comment by mistanoize** [comment]
@Jonta Thank you, I've now resolved 2 of the 4 points thanks to your suggestion. I found some info regarding the Language key/ABC, though nothing that helps. And nothing at all for the 'missing' £ symbol. Should I post these individually in Issues, Discussions, or elsewise?

**Comment by Jonta** [comment]
@mistanoize I'm not here enough these days to be able to say anything you can't see for yourself at a glance

Individually sounds good. =)

**Comment by sslaia** [comment]
To answer your questions:

1. Just choose the bottom row you prefer in the ASK settings: UI > Tweaks and more > Common bottom generic row > Classic (no mic) or choose others you like.
2. In the settings you actually you can hide the top row with that annoying "Dvorak" label or you choose the top row you like: : UI > Tweaks and more > Common top generic row > No top row or choose others you like.
3. I don't understand the question.

**Reply by mistanoize** [reply]
@sslaia Thank you for the help.

Q3 was a matter of selecting Number keys from the Common top generic row options. Thanks to your clear directions for the other queries.

Much obliged.
